subsys/net/lib/openthread/platform/logging.c

```diff
@@ -39,35 +39,14 @@ static inline int log_translate(otLogLevel aLogLevel)
 	return -1;
 }
 
-void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion,
-	       const char *aFormat, ...)
+#if defined(CONFIG_LOG)
+void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, ...)
```

Comment:
Minor detail, but IMHO it would be better to have only one function definition here so
```
void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat, ...)
{
#if defined(CONFIG_LOG)
...
#else 
	ARG_UNUSED(aLogLevel);
	ARG_UNUSED(aLogRegion);
	ARG_UNUSED(aFormat);
#endif 
}
```
This way the function will be always defined in one place even if new parameter is added to it. This way we avoid possible errors if anyone changing forgets to update both definitions. I am not saying that this would happen here, this is more a general comment about the whole code base.

Reply:
or even
```
if (!IS_ENABLED(CONFIG_LOG)) {
  return;
}

Reply:
Thanks for the suggestions, I picked the most code efficient one.